box/box.md

```diff
@@ -1,27 +1,75 @@
 # Realtime Video AI in a Box
 
-## Requirements
+## Usage (Linux AMD64)
+
+```
+export DOCKER=true
+```
+
+1. Start everything with the following command
+```bash
+make box
+```
+2. Start streaming
+```bash
+make box-stream
+```
+
+3. Playback the stream
+```bash
+make box-playback
+```
+
+## Usage (M1 / Linux ARM64)
+
+It requires the following points:
 - go-livepeer compilation configuration (executing `make` should succeed)
 - [mediamtx](https://github.com/bluenviron/mediamtx) is installed (executing `mediamtx` should succeed)
-- [ffmpeg](https://ffmpeg.org/) is installed (executing `ffmpeg` and `ffplay` should succeed)
```

Comment:
don't you still need ffmpeg if not using docker?

Reply:
Yes, added back.